index.js

```diff
@@ -72,6 +72,23 @@ module.exports = {
     "simple-import-sort/imports": "error",
     "simple-import-sort/exports": "error",
     "sort-destructure-keys/sort-destructure-keys": "error",
+    /**
+     * List of sonarjs rules we turned off after release of eslint-plugin-sonarjs v2
+     * Some rules are too restrictive for common use cases,
+     * and others are already covered by other plugins used here.
+     */
+    "sonarjs/different-types-comparison": "off",
+    "sonarjs/function-return-type": "off",
```

Comment:
I'd recommend comments even on rules we turn off, e.g. "This rule is not helpful in TypeScript files, and in JavaScript we often return different types from functions, so this is not a strictness level we want to enforce."